.github/workflows/main.yml

```diff
@@ -160,14 +160,9 @@ jobs:
         run: |
           dotnet restore
           dotnet build
-      - name: Run minicover
+      - name: Run Tests
         run: |
-          dotnet new tool-manifest
-          dotnet tool install --local MiniCover --version 3.4.7
-          dotnet minicover instrument --workdir ./ --assemblies 'Snowflake.Data.Tests/**/bin/**/Snowflake.Data.dll' --sources 'Snowflake.Data/**/*.cs'
-          dotnet minicover reset
```

Comment:
Dont we need this commands to run tests?

Reply:
We only need the `dotnet test` command. We did not use minicover to collect the coverage at all.